src/tests/topp/QCCalculator_2_output.mzQC

```diff
@@ -0,0 +1,123 @@
+{
+  "mzQC": {
```

Comment:
can you check that this file validates successfully using https://github.com/bigbio/mzqc-pylib/tree/master

with something like:
```
from mzqc import MZQCFile as qc
syn_check = qc.SyntacticCheck()
syn_check.validate("{ \"mzQC\": " + qc.JsonSerialisable.ToJson(mzqc) + " }")
```

Reply:
Yes it validates, json.loads() loads the file content into a json object without errors.

Reply:
nice tool, there were some issues that are fixed now and the output validates against this tool